src/selectors/projectLists.js

```diff
@@ -1,7 +1,9 @@
-// @flow
-import type {AppState} from '../types';
-
-export const getProjects = (state: AppState): [Object] => {
+/**
+ * 
+ * @param {AppState} state 
```

Comment:
Should probably replace with `object` or something like that, since AppState is no longer imported.

Reply:
fixed